app/code/community/Oyst/OneClick/Model/Cart.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Cart Model
+ */
+class Oyst_OneClick_Model_Cart
+{
+    public function initOystCheckout($params)
+    {
+        if (!empty($params['add_to_cart_form'])
```

Comment:
For all if conditions, it's better when it's like that for future code modifications in code review
![image](https://user-images.githubusercontent.com/861701/40487185-a37ef81a-5f63-11e8-950d-ce4399527ff8.png)

the modification impact will be only one line.

Same as **,** (comma) for the last element of array :)